src/models/db/subscriptions.js

```diff
@@ -9,12 +9,11 @@ function * getSubscription (id, options) {
   return (yield Subscription.findById(id, options))
 }
 
-function * getMatchingSubscription (subscription, options) {
-  return (yield Subscription.findOne({where: {
+function * getMatchingSubscriptions (subscription, options) {
+  return (yield Subscription.findWhere({
```

Comment:
This should, in principle, return only 1 subscription. See here: https://github.com/interledger/five-bells-ledger/issues/70; and here: https://github.com/interledger/five-bells-ledger/pull/85.